modules/openxr/action_map/openxr_action_map.cpp

```diff
@@ -474,7 +475,93 @@ void OpenXRActionMap::create_default_action_sets() {
-	// TODO implement
+	// Create our Godot action set
+	Ref<OpenXRActionSet> action_set = OpenXRActionSet::new_action_set("godot_editor", "Godot Editor action set");
+	add_action_set(action_set);
+
+	// Create our actions
+	Ref<OpenXRAction> select = action_set->add_new_action("select", "Select", OpenXRAction::OPENXR_ACTION_BOOL, "/user/hand/left,/user/hand/right");
+	Ref<OpenXRAction> alt_select = action_set->add_new_action("alt_select", "Alternative Select (right click)", OpenXRAction::OPENXR_ACTION_BOOL, "/user/hand/left,/user/hand/right");
+	Ref<OpenXRAction> grab = action_set->add_new_action("grab", "Grab", OpenXRAction::OPENXR_ACTION_FLOAT, "/user/hand/left,/user/hand/right");
+	Ref<OpenXRAction> tool_pose = action_set->add_new_action("tool_pose", "Tool pose", OpenXRAction::OPENXR_ACTION_POSE, "/user/hand/left,/user/hand/right");
+
+	// Create our interaction profiles
+	Ref<OpenXRInteractionProfile> profile = OpenXRInteractionProfile::new_profile("/interaction_profiles/khr/simple_controller");
+	profile->add_new_binding(select, "/user/hand/left/input/trigger/click,/user/hand/right/input/trigger/click");
+	// simple controller doesn't support grip or has extra buttons so no alt_select support, shame...
+	profile->add_new_binding(tool_pose, "/user/hand/left/input/aim/pose,/user/hand/right/input/aim/pose");
+	add_interaction_profile(profile);
+
+	// Create our Vive controller profile
+	profile = OpenXRInteractionProfile::new_profile("/interaction_profiles/htc/vive_controller");
+	profile->add_new_binding(select, "/user/hand/left/input/trigger/click,/user/hand/right/input/trigger/click");
+	// no alt_select support, we don't have extra buttons, maybe abuse menu?
+	profile->add_new_binding(grab, "/user/hand/left/input/squeeze/click,/user/hand/right/input/squeeze/click"); // OpenXR will convert bool to float
+	profile->add_new_binding(tool_pose, "/user/hand/left/input/aim/pose,/user/hand/right/input/aim/pose");
+	add_interaction_profile(profile);
+
+	// Create our WMR controller profile
+	profile = OpenXRInteractionProfile::new_profile("/interaction_profiles/microsoft/motion_controller");
+	profile->add_new_binding(select, "/user/hand/left/input/trigger/click,/user/hand/right/input/trigger/click");
+	// no alt_select support, we don't have extra buttons, maybe abuse menu?
+	profile->add_new_binding(grab, "/user/hand/left/input/squeeze/click,/user/hand/right/input/squeeze/click"); // OpenXR will convert bool to float
+	profile->add_new_binding(tool_pose, "/user/hand/left/input/aim/pose,/user/hand/right/input/aim/pose");
+	add_interaction_profile(profile);
+
+	// Create our Meta touch controller profile
+	profile = OpenXRInteractionProfile::new_profile("/interaction_profiles/oculus/touch_controller");
+	profile->add_new_binding(select, "/user/hand/left/input/trigger/click,/user/hand/right/input/trigger/click");
+	profile->add_new_binding(alt_select, "/user/hand/left/input/x/click,/user/hand/right/input/a/click");
```

Comment:
Right click is typically assigned to the b/y button as trigger/x/a are used for the left click.

As mentioned in the comments above, we should also bind the `alt_select` to the touch controller menu button.